app/models/claim.rb

```diff
@@ -23,10 +23,17 @@ class Claim < ApplicationRecord
   belongs_to :school
   belongs_to :provider
 
-  has_many :mentor_trainings
-  has_many :mentors, through: :mentor_trainings
+  has_many(
+    :mentor_trainings,
+    -> { includes(:mentor).order("mentors.first_name", "mentors.last_name") },
```

Comment:
The prototype order the mentors by mentor first name and last name and this is also important when iterating over the mentors to add the hours, they should be ordered by something else than id as uuid ordering is not reliable

Reply:
This would re-use the `Mentor` scope.

```suggestion
    -> { includes(:mentor).merge(Mentor.order_by_full_name) },
```